tools/cli/commands/changelog.js

```diff
@@ -828,50 +806,65 @@ async function promptChangelog( argv, needChangelog ) {
  * Prompts you for how you want changelogger to run (add to all projects or not, etc).
  *
  * @param {object} argv - the arguments passed.
- * @param {Array} needChangelog - files that need changelogs.
+ * @param {Array} defaultProjects - projects that use the default changelog types.
  * @param {Array} uniqueProjects - projects with unique changelog types.
  * @returns {argv}.
  */
-async function changelogAddPrompt( argv, needChangelog, uniqueProjects ) {
-	const totalProjects = [ ...needChangelog, ...uniqueProjects ];
-	if ( uniqueProjects.length === 0 && needChangelog.length > 1 ) {
-		const response = await inquirer.prompt( [
+async function changelogAddPrompt( argv, defaultProjects, uniqueProjects ) {
+	const totalProjects = [ ...defaultProjects, ...uniqueProjects ];
+	let prompts;
+
+	/* changelogConfirm: Add changelog for the one project? */
+	if ( totalProjects.length === 1 ) {
+		prompts = {
+			type: 'confirm',
+			name: 'changelogConfirm',
+			message: `Add changelog for ${ totalProjects[ 0 ] }?`,
```

Comment:
Since we're here, we may as well copyedit the messages too.
```suggestion
			message: `Add a changelog for ${ totalProjects[ 0 ] }?`,
```